src/main/scala/dpla/ingestion3/mappers/providers/JsonRetconMapper.scala

```diff
@@ -0,0 +1,210 @@
+package dpla.ingestion3.mappers.providers
+
+import dpla.ingestion3.mappers.utils.{Document, JsonExtractor, JsonMapping}
+import dpla.ingestion3.model.DplaMapData.{AtLeastOne, ExactlyOne, ZeroToMany, ZeroToOne}
+import dpla.ingestion3.model.{DcmiTypeCollection, DplaPlace, EdmAgent, EdmTimeSpan, EdmWebResource, LiteralOrUri, SkosConcept, URI, eitherStringOrUri, nameOnlyAgent, nameOnlyConcept, nameOnlyPlace, uriOnlyWebResource}
+import org.json4s.JValue
+import org.json4s.JsonAST.{JArray, JObject, JString}
+import org.json4s.JsonDSL._
+import org.json4s.jackson.JsonMethods.{compact, render}
+
+abstract class JsonRetconMapper extends JsonMapping with JsonExtractor {
+
+  // OreAggregation
+  override def dplaUri(data: Document[JValue]): ZeroToOne[URI] = {
+    data.get \ "_source" \ "@id" match {
```

Comment:
`@id` is already the full item path. if you want to construct is use `id` or reuse the entire `@id` prop

```  
_source": {
    "@id": "http://dp.la/api/items/8bca6ea2341791bb4b30c471115e76e8",
    "id": "8bca6ea2341791bb4b30c471115e76e8"
}
```
See 
https://raw.githubusercontent.com/dpla/ingestion3/88eca3ec32ba871ec1de2023a16a36addff557dc/src/test/resources/json-retcon/lc.json